realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -30,13 +30,14 @@
  */
 public class Table implements TableSchema, NativeObject {
 
-    public static final int TABLE_MAX_LENGTH = 56; // Max length of class names without prefix
+    private static final String TABLE_PREFIX = Util.getTablePrefix();
+    public static final int TABLE_NAME_MAX_LENGTH = 62; // Max length of table names
```

Comment:
Thanks. I was thinking the same thing.
In current commits, I kept original value since I wasn't sure if it's just a mistake.

I'll update this PR to use correct value.